includes/breadcrumb.inc

```diff
@@ -24,10 +24,19 @@
  *   not including the given object. For use in the function
  *   drupal_set_breadcrumb().
  */
-function islandora_get_breadcrumbs($object) {
-  $breadcrumbs = array();
```

Comment:
Oops! If I disable breadcrumbs, I get 
Notice: Undefined variable: breadcrumbs in islandora_get_breadcrumbs() (line 44 of /var/www/drupal/htdocs/sites/all/modules/islandora/includes/breadcrumb.inc).

Probably want to leave this line in here.